website/docs/routes/new-media.mdx

```diff
@@ -3483,33 +3483,73 @@ others = 热点新闻 + 滚动新闻
 
 ## 虎嗅 {#hu-xiu}
 
-### 首页资讯 {#hu-xiu-shou-ye-zi-xun}
+### 资讯 {#hu-xiu-zi-xun}
 
-<Route author="HenryQW" example="/huxiu/article" path="/huxiu/article" />
+<Route author="HenryQW nczitzk" example="/huxiu/article" path="/huxiu/article" radar="1" rssbud="1" supportBT="1" supportPodcast="1"/>
```

Comment:
Please remove all `rssbud="1"` in the docs.
a975237f769e56c7d25d9f4e8cc7c8d6e19d2371